nests.json

```diff
@@ -23,6 +23,22 @@
       "category": "application",
       "readme": "https://raw.githubusercontent.com/BEMRosetta/BEMRosetta/master/README.md"
     },
+    {
+      "name": "MoorDyn",
+      "packages": [
+        "MoorDyn_v1",
+		"MoorDyn_v1_DLL",
+        "MoorDyn_v1_5",
+		"MoorDyn_v1_5_DLL",
+		"MoorDyn_v2",
+		"MoorDyn_v2_DLL"
```

Comment:
Do we need all versions? Why not keep only v2?

Reply:
I prefer to maintain them all as v1 is also used by commercial software and v1.5 has the same interface as v1, but internally uses v2. The pace of upgrading of this expensive commercial software is very slow, years late, I don't know why. And of course I want to support it.